test/test_owners.csv

```diff
@@ -809,7 +809,7 @@ k8s.io/kubernetes/pkg/util/tail,zmerlynn,1,
 k8s.io/kubernetes/pkg/util/taints,rrati,0,
 k8s.io/kubernetes/pkg/util/term,davidopp,1,
 k8s.io/kubernetes/pkg/util/threading,roberthbailey,1,
-k8s.io/kubernetes/pkg/util/version,danwinship,0,
+k8s.io/apimachinery/pkg/util/version,danwinship,0,
```

Comment:
I think this file is supposed to be sorted? :man_shrugging:

Reply:
discussed offline, we don't think this file is actually maintained, at the very least this shouldn't break anything.